lockbox-ios/Action/ExternalLinkAction.swift

```diff
@@ -38,6 +38,7 @@ class ExternalLinkActionHandler: ActionHandler {
 
     private func openUrl(string url: String, application: OpenUrlProtocol = UIApplication.shared) {
```

Comment:
unrelated, but while you're in here...

why are you injecting the application twice? it should be sufficient to just use `self.application` in this method, as you've already injected it in `init`